File: cpp/zadaniedomowe_zlozonosc.cpp
#include <iostream>
 
using namespace std;
 
int main(int argc, char **argv)
{
    int n = 0;
    cout<<"Podaj liczbe: ";
    cin>>n;
    int i = 2;
    while(i*i >= n)
    {
        if(n%i != 0)
        {
            cout<<"pierwsza"<<endl;
            break;
        }
        i++;
    }
    while(n%i == 0)
    {
        cout<<"zlozona";
        break;
    }
    return 0;
}

File: python/zadaniedomowe_zlozonosc.py
#!/usr/bin/env python
# -*- coding: utf-8 -*-
 
 
def main(args):
    n = int(input("Podaj liczbe: "))
    i = 2
    while i * i >= n:
        if n % i > 0:
            print("pierwsze")
            break
        i += 1
    while(n % i == 0):
        print("Zlozona")
        break
    return 0
 
 
if __name__ == '__main__':
    import sys
    sys.exit(main(sys.argv))
